### ISSUE TITLE
Trakt Progress: Watched vs Unwatched

### ISSUE THREAD

**Issue by stathis95194** [123Venom/plugin.video.venom#3]
Didn't want to hijack the other issues as i don't have a Trakt Progress issue like that myself. For me it's populating the correct shows, but has watched vs unwatched reversed. So Unwatched has 0 shows which is wrong and Watched has 4 shows with the correct progress which is again wrong. The 2 are reversed.

Kodi 18.2

**Comment by 123Venom** [comment]
can you post a screenshot because I'm not quite following you.  Progress is meant to keep a record of what you've watched to show the next episode to be watched based on that progress.

For example last night I watched Billions S4X05.  My trakt progress shows me 4x06
 because that's what's Next Up to be watched in my progress.  Brings up a point of why I really don't like this called Progress, I think Seren calls it Next Up and makes more sense.  Right before me my wife watched Fringe S2x15, so Progress is showing 2x16 as Next to watch. Since I watched last it's at the top of the list because this was my sort order of choice.  That sort order is something I changed 2 versions ago because my opinion was the default sort order made no sense to me...perhaps that
is what you mean?  That recent changed fixed this ordering for me.

![Capture](https://user-images.githubusercontent.com/46383815/59969527-11fe9d00-951d-11e9-929a-fa3238eb0d2f.PNG)

Here's the revised settings to get it in order based on your Date watched or history order.

![Capture1](https://user-images.githubusercontent.com/46383815/59969535-5b4eec80-951d-11e9-9919-cdfac9470774.PNG)

Is this what you mean?  If not post a case so I can see it.

**Comment by stathis95194** [comment]
Sorry you are right maybe with photos i can explain better.
Here is a photo from My TV Shows: Progress showing my Unwatched episodes/shows. As you can see it's empty
![image](https://user-images.githubusercontent.com/12883036/59973013-e4a80280-95a1-11e9-8c0c-447d57f8e782.png)

Now i change the setting to Watched which is wrong as all the listed episodes are Unwatched. It seems as if the two are reversed.

![image](https://user-images.githubusercontent.com/12883036/59973030-1ae58200-95a2-11e9-8a85-7e785f11e0c1.png)

Not sure if now it's more understandable. Exodus Redux for example has it reversed. And just checked Seren and Next Up that you mentioned and again the 4 episodes are listed as Unwatched, and if i select Watched it's empty.

Hope i make more sense now

**Comment by 123Venom** [comment]
ok to make sure I'm following you correctly....you're going into your Progress and flipping the side panel option to "unwatched"?  I just tested it and you're right it does seem reversed.  I never use that side panel option and just view the lists as "All" so they display the full provided content.  I'm not sure why you'd want to flip that around anyway...why?  Even if it wasn't backwards flipping to "unwatched" should be the same content as "All"...then flipping to "watched" just is going to give you an empty list...why would you want to see an empty list?  I already see why it's happening.  They are being inadvertently marked as watched in a sense, but not actually, it's just a bad indicator call.  If you look close you'll actually see an episode counter in the little box and that's the culprit causing the skin to believe it's watched and getting the check mark, although it's not truly marked...lol..hope you get that meaning.  It's just a visual cue skrew up bbut the skin doesn't know the difference that it's not realy marked.  Redux doesn't suffeer this error because they have no counters in that addon that might have mucked with this.
I'll make a note to look at fixing that but I have a slew of items I'm working on now so might be awhile before I can get to it.  Just leave it on "All" for now.

**Comment by stathis95194** [comment]
No worries at all. I usually keep my indicator to unwatched as I also have local libraries and I need to only see what I have unwatched. As you said not a big deal, just thought to mention it.

**Comment by 123Venom** [comment]
and just so you know.  The counter method/ideas I got from Gaia and just checked Gaia...the progress flipping thing does the same thing there due to the counters.  It can be fixed and I looked at what it would require quickly...a good deal to change to account for those sort of lists so the counters don't mess it up.  Eventually!  Thanks for pointing this out because I actally am now suspicious this is the issue with the empty Progress lists ppl have reported that I literally have about 12hrs into doing dev releases for special logging to see what's going on...lol..happens
